instrumentation/reactor-3.1/library/src/main/java/io/opentelemetry/instrumentation/reactor/TracingOperator.java

```diff
@@ -44,13 +45,20 @@
    */
   public static void registerOnEachOperator() {
     Hooks.onEachOperator(TracingSubscriber.class.getName(), tracingLift());
-    AsyncSpanEndStrategies.getInstance().registerStrategy(ReactorAsyncSpanEndStrategy.INSTANCE);
+    AsyncSpanEndStrategies.getInstance()
+        .registerStrategy(
+            ReactorAsyncSpanEndStrategy.newBuilder()
+                .setCaptureExperimentalSpanAttributes(
+                    Config.get()
+                        .getBooleanProperty(
+                            "otel.instrumentation.reactor.experimental-span-attributes", false))
```

Comment:
Ideally we should only use `Config` in the javaagent... How about adding the builder & flag to `TracingOperator` and making `registerOnEachOperator` an instance method? You won't need a builder for `ReactorAsyncSpanEndStrategy` anymore, you'll be able to use a package-private constructor.

Then in the javaagent code the correct property value will be passed:
```java
TracingOperator.newBuilder()
  .setCaptureExperimentalSpanAttributes(Config.....)
  .build()
  .registerOnEachOperator();
```

Reply:
Got it, will refactor.

Reply:
Thanks! 🙏